web_hosting_manager/app/utils/app_utils.js

```diff
@@ -4,6 +4,29 @@ export const trimErrorMsg = (msg) => {
   return msg.slice(index).trim()
 };
 
+export const parseErrorMsg = (err) => {
+  switch(err.code) {
+    case -102: {
+      return 'Account already exist';
+    }
+    case -107: {
+      return 'Entry already exist';
```

Comment:
As it's shown to the user, it's worthy correcting the missing 's' at the end, i.e. 'Entry already exists'.